flagmodify: Remove the use of flags for all logging options (#5)
uptycslabs/terraform-aws-iam-config: README.md

```
module "iam-config" {
  source          = "github.com/uptycslabs/terraform-aws-iam-config"
  resource_prefix = "cloudquery"

  # These two values are provided by Uptycs
  aws_account_id = "1234567890"
  external_id    = "f09cd4ae-76f1-4373-88da-de721312803d"

>   # Choose one option between cloudtrail bucket and kinesis stream for cloud logs

Reviewer: cloud trail instead of cloudlog

>   # Choose one option between cloudtrail bucket and kinesis stream for cloud logs

Reviewer 2: ```suggestion
  # Choose S3 bucket or Kinesis stream option to ingest CloudTrail 
```

>   # Pass bucket name if customer wants to attach bucket for cloudlogs

Reviewer: cloudtrail

>   # Pass bucket name if customer wants to attach bucket for cloudlogs

Reviewer 2: ```suggestion
  # Provide the S3 bucket name which contains the CloudTrail data
```

  cloudtrail_s3_bucket_name = ""

>   # Pass kinesis stream name if customer wants to attach kinesis stream for cloudlogs

Reviewer: cloudtrail

>   # Pass kinesis stream name if customer wants to attach kinesis stream for cloudlogs

Reviewer 2: ```suggestion
  # Name of the Kinesis stream configured to stream CloudTrail data
```

  kinesis_stream_name = ""

>   # Pass bucket name if customer wants to attach bucket for vpc flow logs

Reviewer 2: ```suggestion
  # Name of the S3 bucket that contains the VPC flow logs
```

| resource_prefix           | Prefix to be used for naming new resources                                                             | `string` | `cloudquery` |
| aws_account_id            | Uptycs AWS account ID                                                                                  | `string` | `""`         |
| external_id               | Role external ID provided by Uptycs                                                                    | `string` | `""`         |
| vpc_flowlogs_bucket_name  | S3 bucket where VPC flow logs are saved. Required, if you want to use S3 bucket for vpc flow logs               | `string` | `""`         |

> | cloudtrail_s3_bucket_name | S3 bucket where CloudTrail is saved. Requried, if you want to use S3 bucket for cloud logs                  | `string` | `""`         |

Reviewer 2: Same as above

> | kinesis_stream_name       | Kinesis stream where CloudTrail logs are streamed. Required, if you want to use kinesis stream for cloud logs | `string` | `""`         |

Reviewer: for cloudtrail

> | kinesis_stream_name       | Kinesis stream where CloudTrail logs are streamed. Required, if you want to use kinesis stream for cloud logs | `string` | `""`         |

Reviewer 2: Replace description with what I suggested above

- The user should have `Administrators` role permission to create resources.
- If the user has multiple aws account profiles then set profile before execute terraform.
  ```sh
    export AWS_PROFILE="< profile name >"
  ```

>  - In file.tf file, choose one option between cloudtrail bucket and kinesis stream for cloudlog monitoring . Give priority to kinesis stream

Reviewer: cloudtrail monitoring

>  - In file.tf file, choose one option between cloudtrail bucket and kinesis stream for cloudlog monitoring . Give priority to kinesis stream

Reviewer 2: ```suggestion
 - In file.tf file, specify CloudTrail S3 bucket name or Kinesis stream name. Kinesis stream based approach provides faster CloudTrail data ingestion
```
